index.html

```diff
@@ -466,347 +466,365 @@ <h3>Ethereum EIP712 Signature 2021</h3>
       </section>
     </section>
 
-    <section>
+    <section id="test-vectors">
       <h2>Test Vectors</h2>
       <p>
         The following test vectors are provided to assist with implementers.
       </p>
 
       <p>
-        The following is an example ECDSA K-256 keypair that can be used to generate EIP712 signatures:
+        The following is an example Ethereum-compatible hexadecimal private key, and corresponding <code>did:pkh</code> <code>verificationMethod</code>  that can be used to assist with test vectors:
       </p>
       <pre class="example">
         {
-          "keypair_0": {
-            "id": "did:example:aaaabbbb#issuerKey-1",
-            "type": "EcdsaSecp256k1VerificationKey2019",
-            "controller": "did:example:aaaabbbb",
-            "publicKeyJwk": {
-              "kty": "EC",
-              "crv": "secp256k1",
-              "x": "cmbYyDC6cbm807_OmFNYP4CLEL0aB2F1UG683SxFkXM",
-              "y": "zBw5HAh0cJM4YimSQvtYM1HFhzUXVUgrDhxJ70aajt0",
-            },
-            "privateKeyJwk": {
-              "kty": "EC",
-              "crv": "secp256k1",
-              "x": "cmbYyDC6cbm807_OmFNYP4CLEL0aB2F1UG683SxFkXM",
-              "y": "zBw5HAh0cJM4YimSQvtYM1HFhzUXVUgrDhxJ70aajt0",
-              "d": "u7QuEl6W0XNppEY0iMVjATT99tC9acwV3Z2keEqvKGo"
-            }
-          }
-        }              
+          "privateKey": "0x149195a4059ac8cafe2d56fc612f613b6b18b9265a73143c9f6d7cfbbed76b7e",
+          "verificationMethod": "did:pkh:eth:0xAED7EA8035eEc47E657B34eF5D020c7005487443"
```

Comment:
```suggestion
          "verificationMethod": "did:pkh:eip155:1:0xAED7EA8035eEc47E657B34eF5D020c7005487443"
```

Reply:
ah woops - will make the change across all test vectors and update their `proofValue` properties as well as these should be reproducible. The draft `did:pkh` spec hasn't yet updated to use CAIP-10 I guess so I just used `did:pkh:eth`

Reply:
yep, for reference: https://github.com/spruceid/ssi/pull/279

Reply:
The fragment part (`#blockchainAccountId`) should also be appended to the verification method id here.

Reply:
Sorry I'm not sure what is meant by `#blockchainAccountId` - does that mean the DID should be something like `did:pkh:eip155:1:0xabc...def#blockchainAccountId` or is that a placeholder for something?

Reply:
Nevermind, got clarification on this. Will add `#blockchainAccountId`